src/elements/Tooltip/Tooltip.tsx

```diff
@@ -58,7 +58,7 @@ enum TipSize {
 }
 
 export interface TooltipProps {
-  content: string
+  content: string | React.ReactElement<any>
```

Comment:
This type should be `React.ReactNode` which includes `string` and other things not covered by `ReactElement`